public/components/datasources/components/manage/connection_configuration.tsx

```diff
@@ -127,7 +126,7 @@ export const ConnectionConfiguration = (props: ConnectionConfigurationProps) =>
           </EuiFormRow>
           <EuiFormRow label="Password">
             <EuiFieldPassword
-              type={dual ? 'dual' : undefined}
+              type={'dual'}
```

Comment:
Minor: We can just keep this`type="dual"`

Reply:
Done